clients/src/main/java/org/apache/kafka/common/requests/LeaderAndIsrRequest.java

```diff
@@ -82,6 +108,10 @@ public LeaderAndIsrRequest build(short version) {
                 data.setIsKRaftController(kraftController);
             }
 
+            if (version >= 5) {
```

Comment:
the field probably should have been ignorable. I guess we shouldn't change it now, though.